broker-core/src/main/java/io/zeebe/broker/engine/EngineServiceNames.java

```diff
@@ -32,6 +32,11 @@
         String.format("%s.rocksdb.storage", partitionName), StateStorageFactory.class);
   }
 
-  public static final ServiceName<ZbStreamProcessorService> ZB_STREAM_PROCESSOR_SERVICE_NAME =
-      ServiceName.newServiceName("logstreams.processor", ZbStreamProcessorService.class);
+  public static final ServiceName<EngineService> ZB_STREAM_PROCESSOR_SERVICE_NAME =
```

Comment:
Maybe we can rename this to ENGINE_SERVICE_NAME so it's consistent with the rest. What do you think?